electron/bridge.ts

```diff
@@ -1,6 +1,7 @@
 import { PaletteMode } from '@mui/material';
 import { contextBridge, ipcRenderer, IpcRendererEvent } from 'electron';
 import { promises as fs } from 'fs';
+import { sep as path_sep } from 'path';
```

Comment:
We don't use snake_case anywhere in this code base - something like `pathSep` would be preferred

Reply:
Done, sorry for that 🙈